presto-main/src/main/java/com/facebook/presto/security/AccessControl.java

```diff
@@ -223,5 +223,5 @@
      *
      * @throws com.facebook.presto.spi.security.AccessDeniedException if not allowed
      */
-    void checkCanSetRole(TransactionId requiredTransactionId, Identity identity, String role, String catalog);
+    void checkCanSetRole(TransactionId requiredTransactionId, Identity identity, String role, String catalogName);
```

Comment:
This isn't strictly needed. But I was trying to keep the signatures of the two methods in `AccessControl.java` and `SystemAccessControl.java` identical, just like all other methods. This can be picked up with other commits that need to be in Andrii's branch.